strongbox-web-core/src/main/java/org/carlspring/strongbox/config/WebConfig.java

```diff
@@ -187,6 +194,21 @@ public Validator localValidatorFactoryBean()
     {
         return new LocalValidatorFactoryBean();
     }
+    
+    @Bean("loggingMgmtDirectoryListingService")
+    public DirectoryListingService getLoggingManagementDirectoryListingService()
+    {
+    	String baseUrl = StringUtils.chomp(configurationManager.getConfiguration().getBaseUrl(), "/");
```

Comment:
```suggestion
        String baseUrl = StringUtils.chomp(configurationManager.getConfiguration().getBaseUrl(), "/");
```

Reply:
Ok, I missed importing the convention XML for eclipse .. :D let me do that and re-format only my code changes.